xradar/io/backends/iris.py

```diff
@@ -3665,9 +3665,15 @@ def get_data_headers(self):
     def site_coords(self):
         """Return Radar Location Tuple"""
         ing_conf = self.ingest_header["ingest_configuration"]
+        lon = ing_conf["longitude_radar"]
+        lat = ing_conf["latitude_radar"]
+        lon = lon if lon <= 180 else lon - 360
+        # todo: is this correct for southern latitudes?
+        lat = lat if lat <= 180 else lon - 360
```

Comment:
I don't think this is right the logic for southern latitudes. Latitude will be between 0 and -90 in the southern hemisphere

Reply:
So, I was thinking the following:

- the output from iris/sigmet file is between 0 and 360 (binary angle decoding)
- the latitude runs from equator (0deg) to the north pole (90deg) and over back to the equator again (180deg, -180deg)
- then latitude runs to the south pole (270deg, -90deg) and up to equator again (0deg)

So with this assumption this formulation would be correct although it looks very weird. I've checked with Py-Art, the same formulation is used there. But no hints in the code.

Reply:
Ahh okay! I am not as familiar with this format... that sounds reasonable then! Sorry for the confusion!

Reply:
No worries, I'm first convinced that this is correct, if I have either proof from file or from Vaisala :grimacing:

Reply:
Sounds good!! Yeah - holding off on merging that Py-ART PR until we get the confirmation.